<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetSetting</class>
 <widget class="QWidget" name="CWidgetSetting">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>用户管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,2,1,1">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="m_cmbFindType">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>系统管理员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>科室管理员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>普通用户</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_lineFindName"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnFind">
           <property name="text">
            <string>查找</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
         <item>
          <widget class="QTableView" name="m_tblvSysUser"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>用户编号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_lineUserID">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>用户类别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_cmbUserType">
             <item>
              <property name="text">
               <string>系统管理员</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>科室管理员</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>普通用户</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>用户名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="m_lineUserName"/>
             </item>
             <item>
              <widget class="QLabel" name="m_lblUserNameTip">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>密码</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="m_lineUserPassword"/>
               </item>
               <item>
                <widget class="QPushButton" name="m_btnShowPassword">
                 <property name="styleSheet">
                  <string notr="true">background-color:transparent</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../Report/Image.qrc">
                   <normaloff>:/Image/eye.ico</normaloff>:/Image/eye.ico</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="m_lblUserPasswordTip">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="m_btnAddUser">
               <property name="text">
                <string>增加</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_btnDelUser">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabServer">
      <attribute name="title">
       <string>服务器材配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>服务器材配置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLineEdit" name="m_lineServer"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblServer">
              <property name="text">
               <string>服务器IP</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="m_lblUsername">
              <property name="text">
               <string>用户名</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>影像位置</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="m_lblPort">
              <property name="text">
               <string>服务器端口</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="m_lblPassword">
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="m_lineDBUsername"/>
            </item>
            <item row="1" column="4">
             <widget class="QLineEdit" name="m_lineDBPassword"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_linePort"/>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="m_lineDicomPos"/>
            </item>
            <item row="1" column="5">
             <widget class="QPushButton" name="m_btnShowDBPassword">
              <property name="styleSheet">
               <string notr="true">background-color:transparent</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Report/Image.qrc">
                <normaloff>:/Image/eye.ico</normaloff>:/Image/eye.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnReset">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnOk">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>数据库网络</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>7</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>标签</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>服务器</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>密码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>database</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>服务器ip</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="3" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="1" column="3" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>标签</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>服务器</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="0" column="4" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>database</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="5" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="0" column="5" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>服务器ip</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Dicom SCP</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>DICOM AE title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IP地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>逻辑名称</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>DICOM AE Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>类型</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IP 地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" alignment="Qt::AlignHCenter">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="1" column="3" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,0,0">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>退出并保存</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Report/Image.qrc"/>
 </resources>
 <connections/>
</ui>
